### PR TITLE
Docs: Headless CMS overview

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds guides for the Headles CMS section and fixes some links related to the topic.
- [Headless CMS overview](https://faststore-site-git-docs-cms-overview-faststore.vercel.app/docs/headless-cms-overview/overview-headless-cms)
- [Part 1: Configuring your VTEX account with the VTEX Headless CMS](https://faststore-site-git-docs-cms-overview-faststore.vercel.app/docs/headless-cms-integration/1-configuring-the-vtex-account)
- [Troubleshooting guide](https://faststore-site-git-docs-cms-overview-faststore.vercel.app/docs/headless-cms-troubleshooting/error-installing-headless-cms)

P.s.: other guides related to the CMS integration will be available in another PR

### CODE REVIEW

File: apps/site/pages/docs/_meta.json [hunk 1]
```diff
@@ -40,6 +40,15 @@
...
     }
   },
+  "-- VTEX Headless CMS --": {
+    "type": "separator",
+    "title": "VTEX Headless CMS"
+  },
+
+  "headless-cms-overview": "Introducing VTEX Headless CMS",
+  "headless-cms-integration": "Integrating your storefront with VTEX Headless CMS",
+  "headless-cms-troubleshooting": "Troubleshooting",
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/_meta.json:50]:
```suggestion
  "headless-cms-overview": {
    "title": "Introducing VTEX Headless CMS"
  },
  "headless-cms-integration": {
    "title": "Integrating your storefront with VTEX Headless CMS"
  },
  "headless-cms-troubleshooting": {
    "title": "Troubleshooting"
  },
```

File: apps/site/pages/index.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { product } from '../mocks/product'
   <section className="nx-hero">
     <div className="nx-hero-wrapper nx-lading-grid">
       <article>
-        <h1>The ultimate toolkit for building blazing-fast storefronts.</h1>
+        <h1>The ultimate toolkit for building blazing-fast storefronts</h1>
```
Comment (Mariana-Caetano) [inline apps/site/pages/index.mdx:16]:
```suggestion
        <h1>The ultimate toolkit for building blazing-fast storefronts.</h1>
```

File: apps/site/pages/docs/headless-cms-overview/overview-headless-cms.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+![CMS Settings](https://vtexhelp.vtexassets.com/assets/docs/src/cms-content-type___ad51b03aa6be86b12b930a6cdb6b3269.png)
+
+### Sections
+
+For each Content Type, different **Sections** will be available to compose that page. Sections represent the content structure of a React component, for example, the [Alert](/components/molecules/alert).
+
+![CMS Section](https://vtexhelp.vtexassets.com/assets/docs/src/cms-sections___ccc8b00d73f222d249a009eb195c8e0c.png)
+
+Sections can be reused in different pages of an ecommerce. You are the one who will choose which Sections will be available at the CMS. Check the following example of the Hero Section being used:
```
Comment (hellofanny) [inline apps/site/pages/docs/headless-cms-overview/overview-headless-cms.mdx:62]:
```suggestion
Sections can be reused in different pages of an ecommerce. You are the one who will choose which Sections will be available at the CMS. Check the following example of the `Hero Section` being used:
```

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+### Step 4 - Communicating WebOps updates to the Headless CMS
+
+Now, if you are developing your FastStore project with CI/CD and VTEX Headless CMS, you must ensure that CI/CD is aware of every CMS update performed via the VTEX Admin. To do so, you must configure the WebOps webhooks responsible for communicating with the VTEX Headless CMS as in the following.
+
+1. Open your FastStore project in any code editor of your preference.
+2. Edit the `faststore.config.js` file in the root directory of your project.
+3. In the VTEX Headless CMS configuration, add the `webhookUrls` property corresponding to your store website as in the following:
+
+   ```js title="faststore.config.js"
```
Comment (hellofanny) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:105]:
```suggestion
   ```js filename="faststore.config.js"
```

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+      webhookUrls: [
+        "https://{account}.myvtex.com/cms-releases/webhook-releases"
+      ]
+    }
+   ```
+
+4. Open a Pull Request in your project including the previous changes.
+5. Merge the Pull Request.
+
+Now you can make changes to the Headless CMS and check if commits are being created in the project. In the following example, the Hero title was modified to `10%`, and we can see that it generated a commit in the repository.
```
Comment (hellofanny) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:116]:
```suggestion
Now you can make changes to the Headless CMS and check if commits are being created in the project. In the following example, the `Hero` title was modified to `10%`, and we can see that it generated a commit in the repository.
```

File: apps/site/pages/docs/headless-cms-overview/overview-headless-cms.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+![CMS Settings](https://vtexhelp.vtexassets.com/assets/docs/src/cms-content-type___ad51b03aa6be86b12b930a6cdb6b3269.png)
+
+### Sections
+
+For each Content Type, different **Sections** will be available to compose that page. Sections represent the content structure of a React component, for example, the [Alert](/components/molecules/alert).
+
+![CMS Section](https://vtexhelp.vtexassets.com/assets/docs/src/cms-sections___ccc8b00d73f222d249a009eb195c8e0c.png)
+
+Sections can be reused in different pages of an ecommerce. You are the one who will choose which Sections will be available at the CMS. Check the following example of the `Hero Section` being used:
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/headless-cms-overview/overview-headless-cms.mdx:62]:
```suggestion
Sections can be reused in different pages of an ecommerce. You are the one who will choose which Sections will be available at the CMS. Check the following example of the [*Hero Section*](/components/organisms/hero) being used:
```

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+<Callout type="warning" emoji="⚠️">
+  Remember to replace the values between curly brackets according to your account name.
+</Callout>
+
+  ```
+  vtex login {account}
+  ```
+
+1. Install the VTEX Headless CMS plugin:
```
Comment (hellofanny) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:45]:
```suggestion
2. Install the VTEX Headless CMS plugin:
```

**Review by hellofanny — APPROVED** [review]
LGTM!

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  If you use Windows as the operating system and find problems while installing the plugin, please refer to our [Troubleshooting](/docs/headless-cms-troubleshooting/error-installing-headless-cms) article.
+</Callout>
+
+
+### Step 2 - Installing the Headless CMS dependencies on your VTEX account
+
+Install the VTEX Headless CMS dependencies in your VTEX account by running the following command:
+
+```bash
+  vtex install vtex.admin-cms-graphql@0.x vtex.admin-cms-graphql-rc@1.x
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:65]:
```suggestion
 vtex install vtex.admin-cms-graphql@0.x vtex.admin-cms-graphql-rc@1.x 
```

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+</Callout>
+
+  ```
+  vtex login {account}
+  ```
+
+2. Install the VTEX Headless CMS plugin:
+
+  ```sh
+   vtex plugins install cms
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:48]:
```suggestion
  vtex plugins install cms
```

File: apps/site/pages/docs/_meta.json
Comment (eduardoformiga) [inline apps/site/pages/docs/_meta.json:1]: hmm, IMO the `VTEX Headless CMS` is a little repeated. We are already under this section. Do you think does it worth to keep it in every page?
Screenshot 2023-07-03 at 18 35 28

File: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+      webhookUrls: [
+        "https://{account}.myvtex.com/cms-releases/webhook-releases"
+      ]
+    }
+   ```
+
+4. Open a Pull Request in your project including the previous changes.
+5. Merge the Pull Request.
+
+Now you can make changes to the Headless CMS and check if commits are being created in the project. In the following example, the `Hero` title was modified to `10%`, and we can see that it generated a commit in the repository.
```
Comment (eduardoformiga) [inline apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx:116]: That's the content from the image below:
```suggestion
Now you can make changes to the Headless CMS and check if commits are being created in the project. In the following example, the `Hero` title was modified to `10% off on first purchase`, and we can see that it generated a commit in the repository.
```

**Review by eduardoformiga — COMMENTED** [review]
Good Job! 
I left some comments along with the review :)

**Review by eduardoformiga — APPROVED** [review]
Amazing job!

**Pull request merged**